projects/storefrontlib/src/cms-components/configurator/commons/config-form/config-form.component.ts

```diff
@@ -73,6 +55,31 @@ export class ConfigFormComponent implements OnInit {
   }
 
   updateConfiguration(event: ConfigFormUpdateEvent) {
+    const owner: GenericConfigurator.Owner = { key: event.productCode };
+
+    //Wait until update is finished
```

Comment:
Will this later be pushed down to the facade layer?

Reply:
I also thought about pushing it to the facade layer (I even had a todo comment in the code to later push it to the facade layer), but I did not do it, because there is no real reuse use case in that case. If you have a different opinion we can discuss this, but I would keep it in the form.

Reply:
I still don't fully understand this. Especially I wonder why the sequence of setGroupStatus and UpdateConfiguration doesn't matter (one is in a subscribe block, the other not). Let's discuss tomorrow, I will schedule a call